components/cards/cards.html

```diff
@@ -266,6 +266,73 @@ <h1>Card Dismiss</h1>
 							sandbox="allow-scripts allow-same-origin"
 						>
 						</iframe>
+
+						<!-- product-card -->
+						<div class="intro-heading lg-text">E-commerce Card</div>
+						<div class="card product-card card-shadow">
+							<a href="#"
+								><img src="../../assets/diary5.jpg" alt="paintings"
+							/></a>
+
+							<div class="price-container text-center">
+								<p class="item-name">Sun Painting</p>
+								<span class="currency">Rs 999.00</span>
+							</div>
+							<a href="#"
+								><button class="btn btn-primary">Add to cart</button></a
+							>
```

Comment:
you can improve code indentation, if it's default by format document then after format document you can manually refactor.